jenkins/job-configs/kubernetes-jenkins/jenkins-janitor.yaml

```diff
@@ -25,4 +25,14 @@
         - activate-gce-service-account
         - shell: |
             python $WORKSPACE/jenkins/janitor.py --project=sen-lu-test --hours=1
+            python $WORKSPACE/jenkins/janitor.py --project=kubernetes-pr-cri-validation --hours=6 --dryrun=true
```

Comment:
Which job shall I refer to to use bootstrap? Seems I don't need a template one.. Is there a document for bootstrap?